File: tests/benchmarks/sort/benchmark.lua
local a = {}

for i = 0, 1000000 do
	table.insert(a, math.random(-10000, 10000))
end

local start = os.clock()

table.sort(a, function(a, b)
	return a > b
end)

io.write(string.format("elapsed: %.8f\n", os.clock() - start))

File: tests/benchmarks/sort/benchmark.py
from __future__ import print_function
import time
import random

# Map "range" to an efficient range in both Python 2 and 3.
try:
    range = xrange
except NameError:
    pass

start = time.clock()
a = []

for i in range(0, 1000000):
    a.append(random.randint(-10000, 10000))

a.sort()
print("elapsed: " + str(time.clock() - start))